usr/src/common/zfs/zfs_prop.c

```diff
@@ -183,6 +196,13 @@ zfs_prop_init(void)
 		{ NULL }
 	};
 
+	static zprop_index_t keystatus_table[] = {
+		{ "none",			ZFS_KEYSTATUS_NONE},
+		{ "unavailable",	ZFS_KEYSTATUS_UNAVAILABLE},
+		{ "available",		ZFS_KEYSTATUS_AVAILABLE},
+		{ NULL }
```

Comment:
the alignment of the ZFS_KEYSTATUS... looks wrong here, but maybe its just github.